content/departments/product-engineering/engineering/cloud/growth-and-integrations/triad-roles-and-responsibilites.md

```diff
@@ -1,20 +1,20 @@
-# Triad Roles and Responsiblities
+# Triad Roles and Responsibilities
 
 ## Definitions
 
 - **Accountable**: The one ultimately answerable for the correct and thorough completion of the deliverable or task, the one who ensures the prerequisites of the task are met, and who delegates the work to those _responsible_.
 - **Responsible**: Those who do the work to complete the task
 - **Expectations**: A description of how we think this should work in practice.
 
-## Roles and Responbilities
+## Roles and Responsibilities
 
 ### Product Strategy
 
 #### Performing research to inform strategy
 
 - **Accountable**: PM
 - **Responsible**: PM
-- **Expectations**: The PM conducts market, customer and quantitative research and synthesizes this into insights that the team can understand.
+- **Expectations**: The PM conducts market, customer, and quantitative research and synthesizes this into insights that the team can understand.
```

Comment:
:( looks like we prefer [Oxford commas](https://handbook.sourcegraph.com/company-info-and-process/communication/content_guidelines/style_and_mechanics/#commas=)

Reply:
I myself love the Oxford comma, but I know it's controversial.

Reply:
Yeah, I guess our corporate Grammarly was configured to use Oxford commas. 
Do you want me to revert all comma changes and leave just typo/rewording fixes? 

P.S: I don't have strong preferences on this)

Reply:
Nope the Oxford comma is in our style guide so keep it.